samples/wcfapp/WcfClient/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.2" />
```

Comment:
I think it would be best to update the samples to be using the latest FX version - 4.7.2